srcpkgs/gnome-remote-desktop/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'gnome-remote-desktop'
 pkgname=gnome-remote-desktop
-version=44.0
+version=45.1
 revision=1
 build_style=meson
-configure_args="-Drdp=true -Dvnc=true -Dsystemd=false
- -Dsystemd_user_unit_dir=/usr/lib/systemd/user"
```

Comment:
Leave this line. We do want to install the systemd service files if possible.

Reply:
fixed